From 833baeb77fcefa1b1fbc6fe3cd00ded6da60ab46 Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 24 Oct 2014 20:28:19 -0400
Subject: [net] openvswitch: remove dup comment in vport.h

Message-id: <86b8b96c8f9695e93707ad1359348cabfa476141.1414182116.git.jbenc@redhat.com>
Patchwork-id: 98449
O-Subject: [RHEL7.1 net PATCH 11/11] net/openvswitch: remove dup comment in vport.h
Bugzilla: 1110384
RH-Acked-by: Flavio Leitner <fbl@redhat.com>
RH-Acked-by: David S. Miller <davem@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110384
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=8159383

commit 8280bf00fdd87de876c4add5fa8ed05c18c78a2f
Author: Wang Sheng-Hui <shhuiw@gmail.com>
Date:   Tue Sep 23 20:02:15 2014 +0800

    net/openvswitch: remove dup comment in vport.h

    Remove the duplicated comment
    "/* The following definitions are for users of the vport subsytem: */"
    in vport.h

    Signed-off-by: Wang Sheng-Hui <shhuiw@gmail.com>
    Acked-by: Pravin B Shelar <pshelar@nicira.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/openvswitch/vport.h b/net/openvswitch/vport.h
index 94f19e8..3e78406 100644
--- a/net/openvswitch/vport.h
+++ b/net/openvswitch/vport.h
@@ -35,7 +35,6 @@ struct vport_parms;
 
 /* The following definitions are for users of the vport subsytem: */
 
-/* The following definitions are for users of the vport subsytem: */
 struct vport_net {
  struct vport __rcu *gre_vport;
 };
-- 
1.7.1